packages/crud/test/crud-buttons.test.js

```diff
@@ -11,6 +12,10 @@ describe('crud buttons', () => {
     fire(crud._grid, 'edit', { item });
   }
 
+  before(async () => {
+    await setViewport({ width: 1024, height: 768 });
```

Comment:
This is needed to workaround #6231. Note, `a11y.test.js` already has the same configuration.